aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -130,6 +135,7 @@ private AWSDataStorePlugin(@NonNull Builder builder) throws DataStoreException {
             SQLiteStorageAdapter.forModels(schemaRegistry, modelProvider) :
             builder.storageAdapter;
         this.categoryInitializationsPending = new CountDownLatch(1);
+        this.reachabilityMonitor = new ReachabilityMonitor();
```

Comment:
Why is this not available to pass in the builder if needed? (e.g. for cleaner tests)

Reply:
Done